skaffold/src/main/kotlin/com/google/container/tools/skaffold/SkaffoldFiles.kt

```diff
@@ -50,13 +51,20 @@ fun isSkaffoldFile(file: VirtualFile): Boolean {
     return false
 }
 
-/**
- * Finds all Skaffold configuration YAML files in the given project.
- *
- * @param project IDE project to search Skaffold file in
- * @return List of Skaffold configuration files in the project.
- */
-fun findSkaffoldFiles(project: Project): List<VirtualFile> {
-    return FileTypeIndex.getFiles(YAMLFileType.YML, GlobalSearchScope.allScope(project))
-        .filter { isSkaffoldFile(it) }
+/** Obtains current active implementation of [SkaffoldFileService] */
+fun getSkaffoldFileService() = ServiceManager.getService(SkaffoldFileService::class.java)!!
```

Comment:
How likely is this service going to be `null`?

Reply:
Not likely in production, but I added that to make sure tests are working fine and I'm replacing the test service right. If it's not likely to be null, do you recommend avoid `!!`?